packages/mask/src/social-network-adaptor/twitter.com/injection/ToolboxHint_UI.tsx

```diff
@@ -48,6 +44,21 @@ export function ToolboxHintAtTwitter(props: { category: 'wallet' | 'application'
             </Text>
         )
     }, [buttonStyle.iconSize, textMarginLeft])
+
+    const [mini, setMini] = useState(false)
+
+    useEffect(() => {
+        const abortController = new AbortController()
+        const watch = new MutationObserverWatcher(searchHomeLinkName()).startWatch(
```

Comment:
CC @Jack-Works

Reply:
I'm not sure if this usage is ok, but since it has a cleanup, maybe it's ok...